### PR TITLE
landscape: add minikube

### PR DESCRIPTION
**Pull request by r2d4**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source?
* [x] If it is open source, does your project have at least 250 GitHub stars?
* [x] Have you picked the single best category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Are you including an SVG or (less preferably) a large PNG?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [ ] 2 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

Few comments:
* I added a new subcategory, local development under "App Definition and Development". Is that category supposed to contain developer tooling? Minikube could also fit under "provisioning", since it does provision a local cluster.

### CODE REVIEW

**Comment by r2d4** [comment]
Thanks! How do I get involved with that effort other than following the issue?

I'll update the PR shortly to put it under app def.

File: landscape.yml [hunk 1]
```diff
@@ -1534,6 +1534,12 @@ landscape:
             logo: ./hosted_logos/kubevirt.svg
             twitter: 'https://twitter.com/kubevirt'
             crunchbase: 'https://www.crunchbase.com/organization/kubevirt'
+          - item:
+            name: Minikube
+            homepage_url: 'https://github.com/kubernetes/minikube'
```
Comment (dankohn) [inline landscape.yml]: We trigger whether it's open source based on the presence of the `repo_url`. You want to do:

```yml
homepage_url: 'https://kubernetes.io/docs/getting-started-guides/minikube/'
repo_url: 'https://github.com/kubernetes/minikube'
```

Comment (r2d4) [reply landscape.yml]: Yup sorry I just added that. It should be in this commit already

**Comment by r2d4** [comment]
LGTM now! I think its ready for a look.

**Pull request merged**